src/libs/Animals/TSeagulls.cpp

```diff
@@ -11,6 +11,7 @@
 //--------------------------------------------------------------------
 TSeagulls::TSeagulls() : enabled(true), count(0), frightened(false)
 {
+    seagulls.reserve(SEAGULL_COUNT);
```

Comment:
consider using resize instead

or use seagulls in initialization list: TSeagulls::TSeagulls() : enabled(true), count(0), frightened(false), seagulls(SEAGULL_COUNT)